src/styles/blocks/page.scss

```diff
@@ -0,0 +1,7 @@
+html {
+  font-family: Roboto, serif;
```

Comment:
Ensure that the font family used here matches the task requirements. Consider defining the font family as a variable in a separate file for maintainability.